frontend/alanda-libraries/projects/alanda-common/src/lib/api/models/task.ts

```diff
@@ -1,29 +1,30 @@
 import { ProjectState } from '../../enums/projectState.enum';
 
 export interface AlandaTask {
-  assignee?: string
-  created?: string
-  due?: string
-  formKey?: string
-  priority?: number
-  description?: string
-  comment?: string
-  pmcProjectGuid?: number
-  candidateGroups?: string[]
-  candidateGroupIds?: number[]
-  state?: ProjectState
-  task_id?: string
-  task_type?: string
-  task_name?: string
-  object_name?: string
-  object_id?: number
-  assignee_id?: string
-  execution_id?: string
-  follow_up?: string
-  process_definition_id?: string
-  process_instance_id?: string
-  process_name?: string
-  process_definition_key?: string
-  process_package_key?: string
-  suspension_state?: boolean
+    assignee?: string;
```

Comment:
The current eslint config does not allow a semi-colon here. Should we change the eslint config so that semi-colon are required? @szavrel 

I am also going to add github CI that checks for linting errors and later on will also run tests

Reply:
Honestly, I would prefer to have semi-colons so I would change the eslint config.

Reply:
@szavrel okey I will create a PR today